jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/tx/DisposeSessionTransactionSynchronization.java

```diff
@@ -42,7 +45,7 @@ public void afterCompletion(int status) {
 	    try {
 	        manager.disposeRuntimeEngine(runtime);
 	    } catch (Throwable e) {
-	        // catch exception as it's only clean up and should not affect runtime
+	        logger.warn("Could not dispose runtime engine", e);
```

Comment:
but isn't mixing manual dispose with automatic dispose something to avoid? I agree that in normal cases it should not fail which is why I initially put it at warn level.